Bottom: f431bd9b47d116c74857f9d0599de67720609ec2
Top:    24d71741d1b1a5f7864b76d2a1114ce850c85250
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 17:21:26 +0100

Refresh of hid-gtk-refactor-viewport-hand

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 1b0b760..067277e 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -135,20 +135,6 @@ GhidGui;
 
 extern GhidGui _ghidgui, *ghidgui;
 
-typedef struct
-{
-  double coord_per_px; /* Zoom level described as PCB units per screen pixel */
-
-  Coord x0;
-  Coord y0;
-  Coord width;
-  Coord height;
-
-  bool flip_x;
-  bool flip_y;
-
-} view_data;
-
   /* The output viewport
    */
 typedef struct
